.github/workflows/integrate.yaml

```diff
@@ -1,37 +1,39 @@
 name: CI
 
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
+  pull_request:
 
 jobs:
   lint:
     name: Lint Check
     runs-on: ubuntu-latest
 
     steps:
-    - name: Check out code
-      uses: actions/checkout@v2
+      - name: Check out code
+        uses: actions/checkout@v2
 
-    - name: Install dependencies
-      run: sudo apt-get install python3-pip tox
+      - name: Install dependencies
+        run: sudo apt-get install python3-pip tox
 
-    - name: Lint code
-      run: tox -e lint
+      - name: Lint code
+        run: tox -e lint
 
   unit:
     name: Unit Test
     runs-on: ubuntu-latest
 
     steps:
-    - name: Check out code
-      uses: actions/checkout@v2
+      - name: Check out code
+        uses: actions/checkout@v2
 
-    - name: Install dependencies
-      run: sudo apt-get install python3-pip tox
+      - name: Install dependencies
+        run: sudo apt-get install python3-pip tox
```

Comment:
I think the pip package for tox is preferred.

Reply:
any particular version preferred? This is my prettier indenting the line. It's still install pip

Reply:
I vote that we leave this for another time since this was just a formatting change